### PR TITLE
Css/initial styling

### PR DESCRIPTION
**Pull request by Andrey-1992**
## Subject:
CSS Styling

## Reason for Pull Request:
Add CSS Styling for main page

## Changes Implemented:
Iteration 0 - CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,45 +1,56 @@
...
       <h2>New Activity</h2>
-      <section class='user-interaction'>
-         <div class='category-type'>
+      <section class="user-interaction">
+         <div class="category-type">
            <h3>Select a category:</h3>
-           <button class='study-cat-btn'>Study</button>
-           <button class='meditate-cat-btn'>Meditate</button>
-           <button class='exercises-cat-btn'>Exercise</button>
+           <section class="buttons">
```
Comment (janitastic) [inline index.html:19]: Add a `flex-wrap: wrap` here in your so the buttons wrap on mobile.

File: index.html [hunk 2]
```diff
@@ -1,45 +1,56 @@
...
+          <form class="user-inputs">
+           <h3>What would you like to accomplish during this time?</h3>
+           <input class="accomplish-task" type="text"/>
+           <div class="times">
+             <section class="min">
+               <h3>Minutes</h3>
+               <input class="min-input" type="text"/>
+             </section>
+             <section class="sec">
+                 <h3>Seconds</h3>
```
Comment (janitastic) [inline index.html:35]: Check your indentation

File: index.html [hunk 3]
```diff
@@ -1,45 +1,56 @@
...
          </div>
-         <div class='user-specs'>
-          <form class='user-inputs'>
-           <label for='accomplish'>What would you like to accomplish during this time?</label>
-           <input class='accomplish-task' type='text'/>
-           <label for='minutes'>Minutes</label>
-           <input class='min-input'/>
-           <label for='seconds'>Seconds</label>
-           <input class='sec-input'/>
+         <div class="user-specs">
```
Comment (LKessell) [inline index.html]: Let's consider changing this div to a section?

File: index.html [hunk 4]
```diff
@@ -1,45 +1,56 @@
...
+          </div>
          </div>
       </section>
      </section>
-     <section class='box2'>
+     <section class="box2">
        <h2>Past Activities</h2>
-       <div class='past-act-msg'>
-         <p>You haven't logged any activities yet</p>
+       <div class="past-act-msg">
```
Comment (janitastic) [inline index.html:48]: This should be a section... anticipating new sections in future iterations.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+/* LEFT SECTION */
+
+.box1 {
+  flex: 2;
+  padding: 20px;
+}
+
+
```
Comment (janitastic) [inline styles.css]: Delete this line

**Pull request merged**